packages/account/src/Sections/Profile/PhoneVerification/PhoneVerificationPage.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './phoneVerification.scss';
+import { LabelPairedArrowLeftCaptionFillIcon } from '@deriv/quill-icons';
+import { Text } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import ConfirmPhoneNumber from './ConfirmPhoneNumber';
+
+type TPhoneVerificationPage = {
+    setShowPhoneVerificationPage: (value: boolean) => void;
+};
+
+const PhoneVerificationPage = ({ setShowPhoneVerificationPage }: TPhoneVerificationPage) => {
+    return (
```

Comment:
```
const PhoneVerificationPage = ({ setShowPhoneVerificationPage }: TPhoneVerificationPage) => (
<div>..</div>)
```

Reply:
was doing it this way was because in future we will handle functions in this page